packages/ckeditor5-paste-from-office/docs/_snippets/features/build-paste-from-office-source.js

```diff
@@ -41,3 +42,4 @@ ClassicEditor.builtinPlugins.push(
 ClassicEditor.defaultConfig.table.contentToolbar.push( 'tableProperties', 'tableCellProperties' );
 
 window.ClassicEditor = ClassicEditor;
+window.ListStyle = ListStyle;
```

Comment:
There's no need to assign new plugins globally. It could be simply added to the `ClassicEditor.builtinPlugins`. See my suggestion above.